README.md

```diff
@@ -1 +1,7 @@
 # Hello-Microverse
+
+This is a project created as part of the Microverse bootcamp.
+
+Author:
+
+David Alvarez
```

Comment:
- [ ] It would be better if you can make your README.md file more   
      professional and descriptive. We may consider 
      using the [Microverse Readme template](https://github.com/microverseinc/readme-template). What do you think?